src/symposium-tickets/symposium-tickets.html

```diff
@@ -116,10 +116,10 @@
       </div>
 
       <div class="field">
-        <label>Full name:</label><input name="name" type="name" />
+        <label>Full name: <input name="name" type="name" /></label>
```

Comment:
By removing the separate label, you should also remove [this line](https://github.com/WISVCH/symposium/pull/58/files/c772214995ce5ae1a7c6b44b6cb2456604e456c3..a1b3374ec20e55158eb236a76bd22f334407d8c7#diff-98c98b2c82c145a8a797299241d880a2R84) which constraints the label to have a minimum width. This line caused the inputs to have the same alignment, but there should be a better solution for this.